### PR TITLE
Addons to current civilian mobs

### PR DESCRIPTION
**Pull request by BioHazardSuit**
#### Summary
Content "Add/adjust some current civilian mobs; added two more types with a gun"

#### Purpose of change
There would be a good chunk of civilians with their sidearms ready despite the firearm and ammo shortage caused by the blob riots + I'd think it'd be more fitting for civilian fighters to arm themselves with random knives in the kitchen/bugout bag instead of a stick they found somewhere.

#### Describe the solution
-Gave the civilian fighter a feral knife monster weapon
-Made a officer civilian that uses an AR15
-Made a civilian fighter that uses a handgun instead
-Added them to last stand map extra

#### Testing
If I remember correctly, everything _should_ work as intended, but

#### Additional context
I made this like a month, maybe a month+ ago and forgot to actually make a pull request. But I'm pretty sure I had fixed around the minor problems back then so it should be good now

### CODE REVIEW

**Comment by Night-Pryanik** [comment]
I'm not a US police weapons expert, but from I've seen on YT, most police officers assault rifles were variations of AR-15, not Steyr AUG.

**Comment by Night-Pryanik** [comment]
Yes, I'd prefer that, but not necessarily specifically AR-15, any variant of it would suffice.

**Comment by BioHazardSuit** [comment]
Alright then. I'll take a look into the police zombie loot table to see what types of AR15s they got first

**Comment by GuardianDll** [comment]
For the idea of PR, ferals (and civilians) do not have ammo intentionally, for two reasons: they would use the rounds long before they met player, and secondly that single gun would would be nearly deadly experience (something we do not emulate now, but want to in the future). Shooting monsters is good, but enemies with guns should be limited to locations where they can have an effectively unlimited supply of such rounds, like bunkers

About PR itself, you have much more changes in this PR that you describe, so you either need to divide it, remove changes that are not related to feral changes, or at least explain what you are doing

**Comment by BioHazardSuit** [comment]
Ok will do. I might have done something to feral guns but i cant tell if thats from my repository or.... Ill try to revert that.

For the civilians with guns, Im thinking theyre designed as scene pieces that die from zombies right? So I'm hoping players don't intentionally aggro on gun wielding civilians cause right, fighting shooting monsters are iffy right now and bullet rng hurts.

And hearing more gunshots at the start of an urban round is immaculate

File: data/json/monster_special_attacks/monster_gun.json [hunk 1]
```diff
@@ -269,6 +269,28 @@
...
     "melee_damage": { "bash": 8 }
   },
+  {
+    "id": "civilian_glock19",
+    "copy-from": "glock_19",
+    "looks_like": "glock_19",
+    "flags": [ "PSEUDO" ],
+    "type": "GUN",
+    "name": { "str_sp": "Fighters' Glock 19" },
+    "description": "A gun carried by civilian monsters, more inaccurate for balancing reasons.",
```
Comment (Procyonae) [inline data/json/monster_special_attacks/monster_gun.json:279]:
```suggestion
    "description": { "str": "A gun carried by civilian monsters, more inaccurate for balancing reasons.  If you see this, it's a bug.", "//~": "NO_I18N" },
```
This prevents stuff like this from being translated, should do the same with the names if they don't show up in game either. (Alternately just don't change the copy-frommed names and descriptions and instead add the reasoning in a "//" comment)

**Comment by Procyonae** [comment]
Personally I'd keep the pistol officer and add a rifle officer sharing their current weight but otherwise you probably want to remove the current pistol item if it's not used elsewhere (it might need migration too idk how monster guns are tracked)

**Comment by BioHazardSuit** [comment]
Ive updated and set my cdda copy again, and i see whats wrong in my repo. I will attempt to fix it later-ish

**Comment by BioHazardSuit** [comment]
Ok looks good. "General build matrix / Basic Build and Test (Clang 10, Ubuntu, Curses) (pull_request) Failing after 1m" isn't something I can fix right? I don't think I can do anything about that

**Pull request merged**